src/CLI/clitree/macro/link_tracking_macro.xml

```diff
@@ -9,7 +9,7 @@
             <PARAM name="upstream" help="Upstream interface" ptype="SUBCOMMAND" mode="subcommand" />
         </PARAM>
         <ACTION>
-            python $SONIC_CLI_ROOT/sonic-cli-link-state-tracking.py set_link_state_tracking_group_upstream ${grp-name} ${iface} ${vlan_id} ${po_name}
+            python $SONIC_CLI_ROOT/sonic_cli_link_state_tracking.py set_link_state_tracking_group_upstream ${grp-name} ${iface} ${vlan_id} ${po_name}
```

Comment:
Please remove this file from your changes.

Reply:
This is actually a bugfix of dell_sonic branch. In dell_sonic, the **link_tracking_macro.xml** points to **sonic-cli-link-state-tracking.py**, which doesn't exist and has been renamed to **sonic_cli_link_state_tracking.py**. So I had to change the name here.